test/collection_test.rb

```diff
@@ -1,12 +1,12 @@
-require 'simplecov'
-SimpleCov.start
-require 'minitest/autorun'
-require 'minitest/pride'
+require_relative './test_helper'
 require 'csv'
 require './lib/collection'
 require './lib/game'
 require './lib/team'
-require './lib/game_team'
```

Comment:
Good call on fixing the names here. Makes it a little easier to understand when we see them in the wild.